<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="hSlider_time">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_nextSong">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>112</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_musicControl">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>112</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_previousSong">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>112</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentTime">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>114</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalTime">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>114</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>/time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_songName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Song</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_singer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>291</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Singer</string>
    </property>
   </widget>
   <widget class="QSlider" name="vSlider_volume">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>22</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_playlist">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>180</y>
      <width>340</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_playType">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>60</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sorted</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_downloadMode">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Download Mode</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
